packages/nx/src/command-line/explain.ts

```diff
@@ -0,0 +1,94 @@
+import * as chalk from 'chalk';
+import { join, relative } from 'path';
+import { nxConfigExplainer } from '../explain/config-explainers/nx-json';
+import { workspaceRoot } from '../utils/app-root';
+import { readJsonFile } from '../utils/fileutils';
+import { output } from '../utils/output';
+
+export interface ExplainArgs {
+  config: string;
+}
+
+export const defaultNxJsonPath = join(workspaceRoot, 'nx.json');
+
+export function explainHandler(args: ExplainArgs) {
+  if (args.config === 'nx.json') {
+    args.config = defaultNxJsonPath;
+  }
+  const resolvedPath = require.resolve(args.config, {
+    paths: [process.cwd()],
+  });
+  const nxJson = readJsonFile(resolvedPath);
+
+  output.log({
+    title:
+      resolvedPath === defaultNxJsonPath
```

Comment:
This should always say 

`Explanation of settings in ${absolutePath}`

Reply:
See comment above regarding absolute path

Reply:
In fact this comment seems to be identical to the first? Marking as resolved to avoid duplication